tests/unit/windows/TestUwbCxAdapterDdiLrpConversion.cxx

```diff
@@ -487,91 +487,113 @@ TEST_CASE("ddi <-> neutral type conversions are stable", "[basic][conversion][wi
-            .Value = rangingInterval,
-        };
         test::ValidateRoundtrip(parameterRangingInterval);
     }
 
     SECTION("UwbApplicationConfigurationParameter enum class variant is stable")
     {
-        // UWB_APP_CONFIG_PARAM_TYPE_AOA_RESULT_REQ
-        constexpr AoAResult aoaResult{ AoAResult::Enable };
-        constexpr UwbApplicationConfigurationParameter parameterAoaResult = {
-            .Type = UwbApplicationConfigurationParameterType::AoAResultRequest,
-            .Value = aoaResult,
-        };
         test::ValidateRoundtrip(parameterAoaResult);
     }
 
     SECTION("UwbApplicationConfigurationParameter std::array<uint8_t, N> variant is stable")
     {
-        // UWB_APP_CONFIG_PARAM_TYPE_STATIC_STS_IV
-        constexpr std::array<uint8_t, StaticStsInitializationVectorLength> staticStsInitializationVector{ 0xAA, 0xBB, 0xCC, 0xDD, 0xEE, 0xFF };
-        constexpr UwbApplicationConfigurationParameter parameterStaticStsInitializationVector = {
-            .Type = UwbApplicationConfigurationParameterType::StaticStsIv,
-            .Value = staticStsInitializationVector,
-        };
         test::ValidateRoundtrip(parameterStaticStsInitializationVector);
     }
 
     SECTION("UwbApplicationConfigurationParameter ::uwb::UwbMacAddress (short type)")
     {
-        // UWB_APP_CONFIG_PARAM_TYPE_DEVICE_MAC_ADDRESS
-        constexpr ::uwb::UwbMacAddress uwbMacAddressShort(std::array<uint8_t, ::uwb::UwbMacAddressLength::Short>{ 0xAA, 0xBB });
-        constexpr UwbApplicationConfigurationParameter parameterUwbMacAddressShort = {
-            .Type = UwbApplicationConfigurationParameterType::DeviceMacAddress,
-            .Value = std::move(uwbMacAddressShort),
-        };
         test::ValidateRoundtrip(parameterUwbMacAddressShort);
     }
 
     SECTION("UwbApplicationConfigurationParameter ::uwb::UwbMacAddress (extended type)")
     {
-        // UWB_APP_CONFIG_PARAM_TYPE_DEVICE_MAC_ADDRESS
-        constexpr ::uwb::UwbMacAddress uwbMacAddressExtended(std::array<uint8_t, ::uwb::UwbMacAddressLength::Extended>{ 0x01, 0x23, 0x45, 0x67, 0x89, 0xAB, 0xCD, 0xEF });
-        constexpr UwbApplicationConfigurationParameter parameterUwbMacAddressExtended = {
-            .Type = UwbApplicationConfigurationParameterType::DeviceMacAddress,
-            .Value = std::move(uwbMacAddressExtended),
-        };
         test::ValidateRoundtrip(parameterUwbMacAddressExtended);
     }
+
+    SECTION("std::vector<UwbApplicationConfigurationParameter>")
+    {
+        const std::vector<UwbApplicationConfigurationParameter> uwbApplicationConfigurationParameters{
+            parameterHoppingMode,
+            parameterNumberOfControlees,
+            parameterSlotDuration,
+            parameterRangingInterval,
+            parameterAoaResult,
+            parameterStaticStsInitializationVector,
+            // parameterUwbMacAddressShort,
+            // parameterUwbMacAddressExtended,

```

Comment:
Test is failing with these included. I'm investigating and will fix prior to merging.

Reply:
Found the bug and working on a fix.

Reply:
FIxed.